src/Paket.Core/FrameworkHandling.fs

```diff
@@ -222,6 +225,7 @@ module FrameworkDetection =
                 | "dnx451" -> Some(DNX FrameworkVersion.V4_5_1)
                 | "dnxcore50" | "netplatform50" | "netcore50" | "aspnetcore50" | "aspnet50" | "dotnet" -> Some(DNXCore FrameworkVersion.V5_0)
                 | v when v.StartsWith "dotnet" -> Some(DNXCore FrameworkVersion.V5_0)
+                | v when v.StartsWith "netstandard" -> Some(DotNetStandard "1.5")
```

Comment:
Yeah I extended it to understand all published standard versions.
On Apr 18, 2016 17:45, "Don Syme" <notifications@github.com> wrote:

> In src/Paket.Core/FrameworkHandling.fs
> <https://github.com/fsprojects/Paket/pull/1605#discussion_r60082429>:
>
> > @@ -222,6 +225,7 @@ module FrameworkDetection =
> >                  | "dnx451" -> Some(DNX FrameworkVersion.V4_5_1)
> >                  | "dnxcore50" | "netplatform50" | "netcore50" | "aspnetcore50" | "aspnet50" | "dotnet" -> Some(DNXCore FrameworkVersion.V5_0)
> >                  | v when v.StartsWith "dotnet" -> Some(DNXCore FrameworkVersion.V5_0)
> > +                | v when v.StartsWith "netstandard" -> Some(DotNetStandard "1.5")
>
> Does this gloss over the differences between "netstandard1.0",
> "netstandard1.3" and "netstandard1.5" and does it mater? thanks
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/fsprojects/Paket/pull/1605/files/299e7083879c167e788152e39d40c59eae01d7e6#r60082429>
>